spec/services/cloudinary_service_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+FILENAME = "moment.jpeg"
+FILE_PATH = "/public/uploads/tmp/moment"
```

Comment:
Can we mock this file path instead of using a real one?

Reply:
Apparently, to successfully upload to cloudinary, I need to attach a real file. I haven't been successful in mocking an image.

You are right about not using a real file path. As a workaround(fairly embarrassed) I created an uploads folder in the specs module. The real image will reside there and we can attach it using the `spec/uploads/` file path.

Pro of this is that this image is reusable in the future. and doesn't reside in a sensitive place.

Not my best work. I am open to jamming on this for a better workaround.

Reply:
No worries and I think what you have is fine! :)